### DISCUSSION TITLE
Self hosted version crippled

### DISCUSSION THREAD

**Discussion by ecxs** [supabase/supabase#10353] [Questions]
Hi there,

I've been playing around with supabase (in the cloud), and today I decided to try out the self hosted version ... only to find out that it is really only a bare bones version. Is this correct, or am I overlooking something?

Thanks in advance, and best regards, Dick

**Comment by ddenev** [comment]
https://github.com/supabase/supabase/discussions/9514

the sad fact is that the owners are silent on the matter, which could be a red flag

**Reply by ecxs** [reply]
Although I liked playing around with Supabase, I've decided to not start using it in production. I've switched to MySQL and am using Sequal Ace on macOS to quickly create databases and tables ...
